tidb-lightning/tidb-lightning-faq.md

```diff
@@ -102,47 +108,21 @@ To stop the `tidb-lightning` process, you can choose the corresponding operation
 
-## Why the `tidb-lightning` process suddenly quits while running in background?
-
-It is potentially caused by starting `tidb-lightning` incorrectly, which causes the system to send a SIGHUP signal to stop the `tidb-lightning` process. In this situation, `tidb-lightning.log` usually outputs the following log:
-
-```
-[2018/08/10 07:29:08.310 +08:00] [INFO] [main.go:41] ["got signal to exit"] [signal=hangup]
-```
-
-It is not recommended to directly use `nohup` in the command line to start `tidb-lightning`. You can [start `tidb-lightning`](/tidb-lightning/deploy-tidb-lightning.md#step-3-start-tidb-lightning) by executing a script.
-
-In addition, if the last log of TiDB Lightning shows that the error is "Context canceled", you need to search for the first "ERROR" level log. This "ERROR" level log is usually followed by "got signal to exit", which indicates that TiDB Lightning received an interrupt signal and then exited.
-
-## Why my TiDB cluster is using lots of CPU resources and running very slowly after using TiDB Lightning?
-
-If `tidb-lightning` abnormally exited, the cluster might be stuck in the "import mode", which is not suitable for production. The current mode can be retrieved using the following command:
-
-{{< copyable "shell-regular" >}}
-
-```sh
-tidb-lightning-ctl --config tidb-lightning.toml --fetch-mode
-```
-
-You can force the cluster back to "normal mode" using the following command:
-
-{{< copyable "shell-regular" >}}
-
-```sh
-tidb-lightning-ctl --config tidb-lightning.toml --fetch-mode
-```
-
 ## Can TiDB Lightning be used with 1-Gigabit network card?
 
-The TiDB Lightning toolset is best used with a 10-Gigabit network card. 1-Gigabit network cards are *not recommended*, especially for `tikv-importer`.
+TiDB Lightning is best used with a 10-Gigabit network card.
 
+<<<<<<< HEAD
 1-Gigabit network cards can only provide a total bandwidth of 120 MB/s, which has to be shared among all target TiKV stores. TiDB Lightning can easily saturate all bandwidth of the 1-Gigabit network and bring down the cluster because PD is unable to be contacted anymore. To avoid this, set an *upload speed limit* in [Importer's configuration](/tidb-lightning/tidb-lightning-configuration.md#tikv-importer):
 
 ```toml
 [import]
 # Restricts the total upload speed to TiKV to 100 MB/s or less
 upload-speed-limit = "100MB"
 ```
+=======
+1-Gigabit network cards can only provide a total bandwidth of 120 MB/s, which has to be shared among all target TiKV stores. TiDB Lightning can easily saturate all bandwidth of the 1-Gigabit network in physical import mode and bring down the cluster because PD is unable to be contacted anymore.
+>>>>>>> 00b23c124 (lightning: split troubleshoot from faq (#10309))
```

Comment:
```suggestion
1-Gigabit network cards can only provide a total bandwidth of 120 MB/s, which has to be shared among all target TiKV stores. TiDB Lightning can easily saturate all bandwidth of the 1-Gigabit network in physical import mode and bring down the cluster because PD is unable to be contacted anymore.
```